dotnet/src/Connectors/Connectors.Memory.Redis/RedisHashSetVectorStoreRecordCollection.cs

```diff
@@ -119,14 +119,14 @@ public RedisHashSetVectorStoreRecordCollection(IDatabase database, string collec
         this._propertyReader.VerifyVectorProperties(s_supportedVectorTypes);
 
         // Lookup storage property names.
-        this._dataStoragePropertyNames = this._propertyReader
-            .DataPropertyStoragePropertyNames
-            .ToArray();
+        var dataStoragePropertyNames = this._propertyReader.DataPropertyStoragePropertyNames;
```

Comment:
Is `this._dataStoragePropertyNames` no longer needed?

Reply:
Yeah, let me get rid of it.